ml-distance-euclidean/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for ml-distance-euclidean v1.0.0
+// Project: https://github.com/mljs/distance-euclidean
+// Definitions by: Diana Caro <https://github.com/CaribeSoy/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'ml-distance-euclidean' {

```

Comment:
Hi and thanks for your comments, these are my first definitions so I'm not sure I understood your request, what do you mean by "strip" (sorry, English is not first language). Since these modules are part of a bigger namespace (ML) I was wondering if I could do something like this (I took [this definition](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/36d40a63a05b5cb3413737fbdf7c9a2a26f211fd/deep-freeze/deep-freeze.d.ts) as an example): 

```
declare namespace ML.Math.Distance.euclidean {
    export interface Euclidean{
        (p: number[], q: number[]): number;
        squared: (p: number[], q: number[]) => number;
    }
}

declare module "ml-distance-euclidean" {
    let  euclidean: ML.Math.Distance.euclidean.Euclidean;
    export = euclidean;
}
```

Or do you mean to get rid of the `declare module "ml-distance-euclidean"` part and export directly the interface and declare `euclidean`?

The usage should be something like this:

```
import * as euclidean from "ml-distance-euclidean";

let v1 = [1, 2, 3, 4];
let v2 = [5, 6, 7, 8];

euclidean(v1, v2);
euclidean.squared(v1, v2);
```